lib/polint/parser.rb

```diff
@@ -0,0 +1,52 @@
+require 'parslet'
+
+module Polint
+  class Parser < Parslet::Parser
+
+    rule(:endl) { str("\n").maybe }
+    rule(:sp) { str(' ') }
+    rule(:htab) { str("\t") }
+    rule(:wsp) { sp | htab }
+    rule(:lwsp) { wsp.repeat }
+    rule(:tcase) { match(/[A-Z]/).repeat(1) >> match(/[a-z]/).repeat }
+    rule(:blank_line) { endl }
+
+    rule(:quote) { str('"') }
+    rule(:quoted_char) { match(/[^"]/) }
+    rule(:quoted_pair) { str('\\') >> quote }
```

Comment:
> Each of untranslated-string and translated-string respects the C syntax for a character string, including the surrounding quotes and embedded backslashed escape sequences

(Spec: https://www.gnu.org/software/gettext/manual/html_node/PO-Files.html)

So... I think we're fine.